website/src/pages/plugins.js

```diff
@@ -167,7 +167,7 @@ function Plugins(props) {
       }
       return (
         <div key={plugin.name}>
-          <PluginCard href={`https://apisix.apache.org/docs/apisix/plugins/${pluginUrl}`} target="_blank">
+          <PluginCard href={`https://apisix.apache.org/docs/apisix/next/plugins/${pluginUrl}`} target="_blank">
```

Comment:
According to mailing list: https://lists.apache.org/thread/fprf95hxjwvbtomndsbd7mh9s7vzfykd

We don't have an agreement on this IMO 🤔